.storybook/pages/ProjectOverview/ProjectOverview.tsx

```diff
@@ -231,7 +231,7 @@ export const ProjectOverview = ({ activeIndex = 0 }: Props) => {
             >
               What to Focus on this Checkpoint
             </Heading>
-            <Text as="div" className="u-margin-bottom-md">
+            <Text as="div" className="mb-4">
               <p>
                 Students individually develop a follow up question that builds
                 from their group’s experiment and describe their follow up
```

Comment:
There's one down on line 243:
```
className="u-margin-bottom-lg"
```

Reply:
Thanks for noticing the class ,it can be replaced with mb-6 tailwind utilities class